src/Entity/CreditMemo.php

```diff
@@ -28,6 +28,9 @@ class CreditMemo implements CreditMemoInterface
     /** @var array */
     private $units;
 
+    /** @var \DateTimeInterface */
+    private $issuedAt;
```

Comment:
We should place it under `$comment` (to keep the order from `__construct`), but it can be easily fixed in next PR